101-automation-with-modules-and-runbook-execution/azuredeploy.json

```diff
@@ -34,44 +34,32 @@
     "Password": {
       "type": "securestring",
       "metadata": {
-        "description": "aduser password"
+        "description": "Azure AD Password to authenticate into automation runbook"
       }
     },
-	 "WebAppName": {
-      "type": "string",
-      "metadata": {
-        "description": "This is the name of the web app that may be provided dynamically after creating."
-      }
-    },
-	 "WebAppResourceGroup": {
+    "jobId": {
       "type": "string",
       "metadata": {
-        "description": "This is the name of the resource group in which web app resides."
+        "description": "The GUID for the runbook job to be started."
       }
     },
-    "jobId": {
+    "SubscriptionId": {
       "type": "string",
       "metadata": {
-          "description": "The GUID for the runbook job to be started."
+        "description": "It is the Id of the subscription in which webapp resides"
       }
-  },
-  "SubscriptionName":{
-    "type": "string",
-  "metadata": {
-      "description": "SubscriptionName"
-  }
-  }
+    }
   },
   "variables": {
     "automationaccountname": "[concat('automation', uniquestring(resourceGroup().id))]",
     "runbookName": "initialsetup",
-    "scriptUri": "https://storageformarketplace.file.core.windows.net/cblaze/1.0.0/initialSetup.ps1?sv=2017-11-09&ss=f&srt=sco&sp=r&se=3018-09-18T20:44:18Z&st=2018-09-18T12:44:18Z&spr=https,http&sig=1zNq%2FUzgU937NgY%2BmpqJaWXEfGqTncQ5UMdypkqxLPE%3D"
+    "scriptUri": "https://storageformarketplace.file.core.windows.net/cblaze/1.0.0/testinitialSetup.ps1?sv=2017-11-09&ss=f&srt=sco&sp=r&se=3018-09-18T20:44:18Z&st=2018-09-18T12:44:18Z&spr=https,http&sig=1zNq%2FUzgU937NgY%2BmpqJaWXEfGqTncQ5UMdypkqxLPE%3D"
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#deployment-artifacts-nested-templates-scripts) for how to handle artifacts